docs/threat-actor-intel.md

```diff
@@ -0,0 +1,70 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Threat Actor Intelligence
+
+The **Threat Actor Intelligence** feature in Threat Intelligence Management (TIM) provides analysts with detailed insights into adversaries, their motivations, and their tactics, techniques, and procedures (TTPs).
+
+## What is Threat Actor Intelligence
+
+A *threat actor* is an individual, group, or organization responsible for conducting malicious cyber activities. FortiGuard provides continuously updated profiles of threat actors that include their capabilities, targeted sectors, geographical scope, and known campaigns.
+
+By integrating this intelligence into TIM, SOC analysts gain enriched context that goes beyond indicators or feeds—linking observed activity to known adversaries and their historical behaviors.
+
+## How Threat Actor Data is Used in TIM
+
+- **Correlation with Threat Reports**: TIM automatically correlates FortiGuard threat actor data with existing feeds, reports, and ingested observables. Analysts can view connections between threat actors and ongoing campaigns, giving greater context to alerts.
+
+- **Mapping to MITRE ATT&CK**: Threat actor profiles are correlated with MITRE ATT\&CK data, updating TIM fields such as *techniques*, *sub-techniques*, and *tactics*. This mapping enables SOC analysts to quickly identify likely attack patterns and understand potential adversary objectives.
+
+- **Seamless Integration**: The threat actor module is already bundled in TIM. Retrieved information is automatically mapped to existing TIM fields, including:
+
+    - **Actor profile**: Name, aliases, first seen/last seen.
+    - **Target sectors and regions**
+    - **Associated malware and tools**
+    - **Observed techniques and sub-techniques (MITRE mapping)**
+    - **Related indicators and campaigns**
+
+## Launching Threat Actor Intelligence
+
+To access threat actor intelligence in TIM:
+
+1. In the left navigation pane, go to **Threat Intel Management** <picture><source media="(prefers-color-scheme: dark)" srcset="./res/icon-chevron-light.svg"><source media="(prefers-color-scheme: light)" srcset="./res/icon-chevron-dark.svg"><img alt="Fallback image description" src="./res/icon-chevron-dark.svg"></picture> **Threat Actor**.
+
+2. Cards containing threat actor information are displayed.
+
+    ![Threat Actor Intelligence](./res/threat-actor-dashboard.png)
```

Comment:
Image is not rendering

Reply:
<img width="1091" height="615" alt="image" src="https://github.com/user-attachments/assets/a3db2ad1-b45c-42b0-8030-e01399754737" />